lib/features/common/domain/entity/tokens/tokens_entity.dart

```diff
@@ -0,0 +1,13 @@
+/// {@template tokens_entity.class}
+/// Tokens entity
+/// {@endtemplate}
+final class TokensEntity {
```

Comment:
This is more of a philosophical question about architecture, but why are tokens considered entities? It hardly seems that the presence of tokens relates to the application's domain.

Reply:
refactoring